src/xtgeo/cube/_cube_import.py

```diff
@@ -20,103 +48,105 @@ def import_segy(sfile: xtgeo._XTGeoFile) -> Dict:
     """Import SEGY via the SegyIO library.
 
     Args:
-        sfile (str): File name of SEGY file
+        sfile: File object for SEGY file
     """
-    # pylint: disable=too-many-statements
-    # pylint: disable=too-many-locals
     sfile = sfile.file
 
-    logger.debug("Inline sorting %s", segyio.TraceSortingFormat.INLINE_SORTING)
+    attributes = dict()
+
+    try:
+        # cube with all traces present
+        with segyio.open(sfile, "r") as segyfile:
+            attributes = _import_segy_all_traces(segyfile)
+    except ValueError as verr:
+        if any([word in str(verr) for word in ["Invalid dimensions", "inconsistent"]]):
+            # cube with missing traces
+            print(
+                "XTGeo notice: Missing or inconsistent traces in SEGY, will try "
+                "but check result carefully!"
+            )
```

Comment:
I would advise against this, would rather raise an exception saying something like: 
```
"XTGeo notice: Missing or inconsistent traces in SEGY, rerun with incomplete=True to attempt loading anyway
```
  and expose `incomplete` in the function.

Reply:
Discussed this a bit offline, but still leaning towards this being an error, or at least a warning. Espesially if it does not occur often.

Reply:
I think you are right; I have changed it to a warning